it.mijorus.gearlever.json

```diff
@@ -0,0 +1,73 @@
+{
+    "app-id" : "it.mijorus.gearlever",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "44",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "gearlever",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=wayland",
+        "--socket=fallback-x11",
+        "--device=dri",
+
+        "--filesystem=xdg-data/applications:rw",
```

Comment:
Is this to create a new desktop file? There's a portal for that.

Reply:
Yes, I tried using the portal but I didn't manage to get it working.

Do you know another app that is using it? Maybe I can see their implementation...

Reply:
Btw, Gear lever can also manage apps that have been integrated with other tools, so this permission is still necessary

Reply:
An example in Rust would be this: https://docs.rs/ashpd/0.4.0/ashpd/desktop/dynamic_launcher/index.html#examples and I found another one here: https://github.com/flatpak/xdg-desktop-portal/issues/826